### ISSUE TITLE
Emoji incorrectly identified as trailing punctuation

### ISSUE THREAD

**Issue by damies13** [github/super-linter#3982]
## Describe the bug

Emoji is incorrectly identified as trailing punctuation

## Expected behavior

MD026 should identify that a `:` character that is part of an emoji is not punctuation, expect not to get this error

## Steps to Reproduce

Add this line to an .md file and run super-linter

```
> #### :warning: Warning :warning: 
```
> #### :warning: Warning :warning: 

## Logs

```
<filename>.md:193:34 MD026/no-trailing-punctuation Trailing punctuation in heading [Punctuation: ':']

```

**Comment by damies13** [comment]
In case anyone else encounters this issue, this was my workaround:

```
> #### :warning: Warning :warning: <!-- this is here for super-linter -->
```

**Comment by damies13** [comment]
I know this is a minor bug that only happens in specific circumstances and I've provided a workaround, but I would appreciate if someone looked at it to decide if it could be fixed?

**Comment by ferrarimarco** [comment]
This should be reported against Markdownlint, not here.
